tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -988,9 +986,16 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                               default=())
     # TODO: isn't it a weights precision type
     common_group.add_argument('--data_type',
-                              help=mo_convert_params['data_type'].description,
+                              help='[DEPRECATED] Data type for all intermediate tensors and weights. '
+                                   'If original model is in FP32 and --data_type=FP16 is specified, '
+                                   'all model weights and biases are compressed to FP16.',
                               choices=["FP16", "FP32", "half", "float"],
-                              default='float')
+                              default='float',
+                              action=DeprecatedOptionCommon)
+    common_group.add_argument('--convert_to_fp16',
```

Comment:
Should be `compress_to_fp16`, not `convert_to_fp16`

Reply:
Done.